consensus/types/src/chain_spec.rs

```diff
@@ -1701,9 +1669,6 @@ impl Config {
             maximum_gossip_clock_disparity_millis: spec.maximum_gossip_clock_disparity_millis,
             message_domain_invalid_snappy: spec.message_domain_invalid_snappy,
             message_domain_valid_snappy: spec.message_domain_valid_snappy,
-            attestation_subnet_extra_bits: spec.attestation_subnet_extra_bits,
```

Comment:
The removal of these fields may break some third-party HTTP API consumers. I will flag this PR as backwards-incompat so that this is mentioned in the release notes.